radionets/evaluation/plotting.py

```diff
@@ -734,17 +799,23 @@ def histogram_peak_intensity(ratios_peak, out_path, plot_format="png"):
         alpha=0.75,
     )
     ax1.axvline(1, color="red", linestyle="dashed")
-    ax1.set_xlabel("Ratio of intensity peak flux")
+    ax1.set_xlabel("Ratio of intensity peak fluxes")
```

Comment:
I think this should be flux densities as our source images are in units of Jy/pixel, which is a flux density

Reply:
Done